### PR TITLE
docs: improve the `README.md` file

### PR DESCRIPTION
**Pull request by Panquesito7**
### Description

- Improve the `README.md` file.

### Code of Conduct

- [x] By submitting this pull request, I confirm I've read and complied with the [CoC](https://github.com/dev-protocol/community/blob/main/CODE_OF_CONDUCT.md) 🖖

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,39 +10,44 @@
   </a>
 </p>
 
-![Dev Protocol](https://user-images.githubusercontent.com/73097560/126608267-c422adf1-d201-4ef4-ad40-502dc69c4f6f.png)
+![Banner](https://user-images.githubusercontent.com/73097560/126608267-c422adf1-d201-4ef4-ad40-502dc69c4f6f.png)
```
Comment (krishguptadev) [inline README.md:13]:
```suggestion
[![Banner](https://user-images.githubusercontent.com/73097560/126608267-c422adf1-d201-4ef4-ad40-502dc69c4f6f.png)](https://devprotocol.xyz)
```

File: README.md [hunk 2]
```diff
@@ -10,39 +10,44 @@
...
 
-## Why you should join this community?
+## 🤔 Why you should join this community?
+
 - Accelerate your learning by contributing with our dApps open-source
 - Get interact more with Web3 and Blockchain
 - Interact with great members
 
-[Join now](https://github.com/dev-protocol/community/issues/new?assignees=&labels=github-invitation&template=invitation.yml&title=Please+invite+me+to+the+community) | [Explore](https://github.com/dev-protocol)
+### [Join now](https://github.com/dev-protocol/community/issues/new?assignees=&labels=github-invitation&template=invitation.yml&title=Please+invite+me+to+the+community) | [Explore](https://github.com/dev-protocol)
```
Comment (krishguptadev) [inline README.md:25]:
```suggestion
**[Join now](https://github.com/dev-protocol/community/issues/new?assignees=&labels=github-invitation&template=invitation.yml&title=Please+invite+me+to+the+community) | [Explore](https://github.com/dev-protocol)**
```

Comment (Panquesito7) [reply README.md:25]: IMO, it looks better using `###` as it makes the letters bigger.

**Review by krishguptadev — APPROVED** [review]

**Review by vinzvinci — COMMENTED** [review]
Awesome! LGTM 👍

**Pull request merged**